frontend/src/pages/MainPage/index.js

```diff
@@ -82,6 +101,16 @@ const MainPage = () => {
         const data = await response.json();
 
         setPosts(data);
+
+        const pageParams = queryString.stringify(postQueryParams);
+        const filterParams = selectedFilterDetails
+          .map((filter) => {
+            return filter.filterType + '=' + filter.filterDetailId;
+          })
+          .join('&');
```

Comment:
```suggestion
        const filterParams = selectedFilterDetails
          .map((filter) => filter.filterType + '=' + filter.filterDetailId)
          .join('&');
```
브라운은 template literal 을 별로 선호하지 않으시군요..! 
여기 map 부분의 return은 없애면 가독성에 더 좋을 것 같아여!